src/main/java/net/snowflake/ingest/connection/RequestBuilder.java

```diff
@@ -565,25 +565,23 @@ private static void addUserAgent(HttpUriRequest request, String userAgentSuffix)
    * addToken - adds a token to a request
    *
    * @param request the URI request
-   * @param token the token to add
    */
-  private void addToken(HttpUriRequest request, String token) {
-    request.setHeader(HttpHeaders.AUTHORIZATION, BEARER_PARAMETER + token);
+  public void addToken(HttpUriRequest request) {
+    request.setHeader(HttpHeaders.AUTHORIZATION, BEARER_PARAMETER + securityManager.getToken());
```

Comment:
Trying to understand why the code was written this way before, can this token be updated in securityManager after it's passed into addToken?